src/internal.c

```diff
@@ -20466,10 +20420,9 @@ int TimingPadVerify(WOLFSSL* ssl, const byte* input, int padLen, int macSz,
 
 int DoApplicationData(WOLFSSL* ssl, byte* input, word32* inOutIdx, int sniff)
 {
-    word32 msgSz   = WOLFSSL_IS_QUIC(ssl)? ssl->curSize : ssl->keys.encryptSz;
```

Comment:
why are we changing QUIC specific behaviour here?

Reply:
Because now we want to use `ssl->curSize` in all cases.